Task2_2.py

```diff
@@ -0,0 +1,38 @@
+import string
+import random
+
+# creating list of dicts
+list_of_dicts = []
+# set number of dicts
+for k in range(random.randint(2, 10)):
+    # add random keys and values and number of key-value pairs
+    mydict = {chr(random.randint(ord('a'), ord('z'))): random.randint(0, 100) for i in range(random.randint(1, 100))}
+    # add dicts to the list of dicts
+    list_of_dicts.append(mydict)
+print('list of dicts:', list_of_dicts)
+
+#creating temporary dict
+tmp_dict = {}
+#getting numbers of list elements
+for i, d in enumerate(list_of_dicts):
+    #iterating each dict in list
+    for k, v in d.items():
+        if k not in tmp_dict:
```

Comment:
As you can see here using such variables as i, d, k, v complicate readability of your code for others, it's really hard to catch already what is what here

Reply:
```suggestion
for index, dict in enumerate(list_of_dicts):
    #iterating each dict in list
    for letter, value in dict.items():
        if letter not in tmp_dict:
```